From 34124bf064dd9ac2515f8a501474e9bb61d7a7d9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 10 Jun 2020 11:39:41 +0000
Subject: [PATCH] Fix AppStream metadata file

Fix license entries to meet the specification, and update homepage URL.
---
 misc/unix/grafx2.appdata.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/misc/unix/grafx2.appdata.xml b/misc/unix/grafx2.appdata.xml
index 2f683d87..7a847e80 100644
--- a/misc/unix/grafx2.appdata.xml
+++ b/misc/unix/grafx2.appdata.xml
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id type="desktop">grafx2.desktop</id>
-  <metadata_licence>CC-0</metadata_licence>
-  <project_license>GPL-2.0 and CeCILL-2</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0 and CECILL-2.0</project_license>
   <name>GrafX2</name>
   <summary>A bitmap paint program specialized in 256 color drawing</summary>
 
@@ -31,7 +31,7 @@
     </screenshot>
   </screenshots>
 
-  <url type="homepage">http://grafx2.tk</url>
+  <url type="homepage">http://grafx2.chez.com/</url>
   <url type="bugtracker">https://pulkomandy.tk/projects/GrafX2</url>
   
   <provides>
-- 
2.27.0

